packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -261,290 +252,27 @@ function ItemsPerPageControl() {
-					// opens and the focus goes there the actions would be hidden.
-					// To avoid that we add a class to the item, that makes sure actions are visible while there is some
-					// interaction with the item.
-					{ 'is-interacting': isChangingPreviewOption }
-				) }
-				justify="flex-start"
-			>
-				<span className="dataviews-field-control__icon">
-					{ ! canMove && ! field.enableHiding && (
-						<Icon icon={ lock } />
-					) }
-				</span>
-				<span className="dataviews-field-control__label-sub-label-container">
-					<span className="dataviews-field-control__label">
-						{ label || field.label }
-					</span>
-					{ description && (
-						<span className="dataviews-field-control__sub-label">
-							{ description }
-						</span>
-					) }
-				</span>
-				<HStack
-					justify="flex-end"
-					expanded={ false }
-					className="dataviews-field-control__actions"
-				>
-					{ isVisible && (
-						<>
-							<Button
-								disabled={ isFirst || ! canMove }
-								accessibleWhenDisabled
-								size="compact"
-								onClick={ onMoveUp }
-								icon={ chevronUp }
-								label={
-									isFirst || ! canMove
-										? __( "This field can't be moved up" )
-										: sprintf(
-												/* translators: %s: field label */
-												__( 'Move %s up' ),
-												field.label
-										  )
-								}
-							/>
-							<Button
-								disabled={ isLast || ! canMove }
-								accessibleWhenDisabled
-								size="compact"
-								onClick={ onMoveDown }
-								icon={ chevronDown }
-								label={
-									isLast || ! canMove
-										? __( "This field can't be moved down" )
-										: sprintf(
-												/* translators: %s: field label */
-												__( 'Move %s down' ),
-												field.label
-										  )
-								}
-							/>
-						</>
-					) }
-					{ onToggleVisibility && (
-						<Button
-							className="dataviews-field-control__field-visibility-button"
-							disabled={ ! field.enableHiding }
-							accessibleWhenDisabled
-							size="compact"
-							onClick={ () => {
-								onToggleVisibility();
-								focusVisibilityField();
-							} }
-							icon={ isVisible ? unseen : seen }
-							label={
-								isVisible
-									? sprintf(
-											/* translators: %s: field label */
-											_x( 'Hide %s', 'field' ),
-											field.label
-									  )
-									: sprintf(
-											/* translators: %s: field label */
-											_x( 'Show %s', 'field' ),
-											field.label
-									  )
-							}
-						/>
-					) }
-					{ previewOptions && (
-						<PreviewOptions
-							previewOptions={ previewOptions }
-							onChangePreviewOption={ onChangePreviewOption }
-							onMenuOpenChange={ setIsChangingPreviewOption }
-							activeOption={ field.id }
-						/>
-					) }
-				</HStack>
+		<Item onClick={ field.enableHiding ? onToggleVisibility : undefined }>
```

Comment:
We can revise later if it makes sense to extract something from dropdown menu with checkboxes, which behaves similar to this

Reply:
Sounds like a Listbox :D https://github.com/WordPress/gutenberg/issues/65801